dart/lib/flat_buffers.dart

```diff
@@ -109,6 +109,10 @@ abstract class ObjectBuilder {
 class Builder {
   final int initialSize;
 
+  /// Finished buffer size.
+  /// Null until [finish] or [lowFinish] is called.
+  int size;
```

Comment:
This should probably be get only publically, and settable internally. e.g.

```
int _size;
int get size => _size;
```

Reply:
done